library/agent/ServiceConfig.ts

```diff
@@ -127,6 +128,21 @@ export class ServiceConfig {
     this.setBlockedIPAddresses(blockedIPAddresses);
   }
 
+  updateBlockedUserAgents(blockedUserAgents: string) {
+    if (!blockedUserAgents) {
+      this.blockedUserAgentRegex = undefined;
+      return;
+    }
+    this.blockedUserAgentRegex = new RegExp(blockedUserAgents, "i");
```

Comment:
This might throw?

Reply:
The location where this method is called is already surrounded by a try/catch.